src/web/app/src/components/SignUp/Forms/Review.tsx

```diff
@@ -106,6 +106,36 @@ const useStyles = makeStyles((theme: Theme) =>
 const Review = connect<{ accountError: string | undefined }, SignUpForm>((props) => {
   const classes = useStyles();
 
+  // Make sure only confirmed channel URL(s) are accepted, after data is "submitted" by "Youtube and Twitch" pages
+  const channelList = props.formik.values.channelUrl;
+  if (channelList.trim() !== '') {
+    // If there is any confirmed channel, match the URL(s) against channel URL list to eliminate unconfirmed ones from the channel URL list
+    const confirmedChannels = props.formik.values.channels;
+    if (confirmedChannels.length > 0) {
+      const confirmedChannelUrls = confirmedChannels.map((channel) => {
+        return channel.feedUrl;
+      });
+
+      let channelListArray = channelList.split(/ +/);
+      // This skips any youtube channel URL at the moment
+      channelListArray = channelListArray.filter((channel) =>
+        confirmedChannelUrls.find(
```

Comment:
I did it this way because I am matching against part of the string in each element of `confirmedChannelUrls`. I believe `.includes()` for arrays requires an exact match of the element, right?
I first did it with `.includes()` as you suggest, but that version introduced 2 changes that needed to be made to skip YouTube URLs, which I figure would be messier to keep track of if any future developer wishes to change it.